### PR TITLE
add message log to sharp

### PR DESCRIPTION
**Pull request by FrostbiteDragon**

### CODE REVIEW

File: DiscordBot/Modules/ConfigModule.cs [hunk 1]
```diff
@@ -34,4 +34,16 @@ public async Task SetPrefix(char prefix)
...
+	[Command("enable messagelog")]
+	[Summary("set prefix _prefix_\n sets the command prefix")]
+	[RequireUserPermission(ChannelPermission.ManageMessages)]
+	public async Task EnableMessageLog(bool value)
+	{
+		await _config.SetMessageLog(Context.Guild, value);
+
+        if (value)
+            await ReplyAsync($"Message log enabled");
+		else
```
Comment (ZachBaird) [inline DiscordBot/Modules/ConfigModule.cs:46]: Is the indenting here weird?

File: SharpBot.Data/GuildService.cs [hunk 2]
```diff
@@ -38,5 +38,16 @@ public void SetPrefix(ulong guildId, char prefix)
...
+
+        public void SetMessageLog(ulong guildId, bool value)
+        {
+            using var connection = new NpgsqlConnection(connectionString);
+            connection.Open();
+            connection.Execute($"select set_message_log('{guildId}', {value})");
+
+            if (value)
+                try { connection.Execute($"select create_message_partition('{guildId}')"); }
+                catch { }
```
Comment (ZachBaird) [inline SharpBot.Data/GuildService.cs:50]: I know we talked about empty catches and fixing em later, I'm just pointing it out again so it doesn't drop off our radar.

Comment (FrostbiteDragon) [reply SharpBot.Data/GuildService.cs:50]: yep, i put it on trello

File: DiscordBot/Modules/ConfigModule.cs (hunk 1 above)
Comment (FrostbiteDragon) [reply DiscordBot/Modules/ConfigModule.cs:46]: not on my local... and I don't have anything to push either

**Pull request merged**